### PR TITLE
[java] fix highlight for ":" after label

### PR DESCRIPTION
**Pull request by p00f**

### CODE REVIEW

**Comment by p00f** [comment]
the previous commit (792c87c from #302) is somehow included too

File: queries/java/highlights.scm [hunk 1]
```diff
@@ -216,5 +178,47 @@
 ] @punctuation.bracket
 
 ; Labels
+
 (labeled_statement
   (identifier) @label)
```
Comment (stsewd) [inline queries/java/highlights.scm:183]: You could also match the delimiter here, but not strong opinion.

**Comment by stsewd** [comment]
> the previous commit (792c87c from #302) is somehow included too

That's because you are using the master branch, you should create a separate branch for each pull request :)

**Comment by p00f** [comment]
Going offline for like a month, will do this later

**Pull request closed without merging**